src/ol/control/control.js

```diff
@@ -121,3 +124,18 @@ ol.control.Control.prototype.setMap = function(map) {
     map.render();
   }
 };
+
+
+/**
+ * This function is used to set a target element for the control. This
+ * function will have no effect if it is called after the control has
```

Comment:
I think this is a limitation which is not necessary. With a little more code, you could use setTarget to move the control's markup around in the document. Or am I missing something?